docs/source/index.md

```diff
@@ -56,9 +56,14 @@ contributing/contributing
 ---
 maxdepth: 3
 name: modules
-caption: Modules
+caption: Torchsynth Packages
 ---
-modules
+defaults
+filter
```

Comment:
Same here, remove filters

Reply:
So I can remove filter.md from the project. Right?

Reply:
Yep, go ahead, thank you

Reply:
Done